package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "todo_challenge",
+  "version": "1.0.0",
+  "description": "* Deadline: submit completed pull request by 9am on Monday * You may use whatever level of JavaScript you feel comfortable with - pure JS, jQuery, Angular, or whatever weird and wonderful framework you want to try. Extra points for DogeScript",
+  "main": "index.js",
+  "scripts": {
+    "test": "protractor conf.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ErikAGriffin/todo_challenge.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ErikAGriffin/todo_challenge/issues"
+  },
+  "homepage": "https://github.com/ErikAGriffin/todo_challenge",
+  "dependencies": {
+    "express": "^4.12.3"
+  },
+  "devDependencies": {
+    "jasmine-core": "^2.2.0",
+    "karma": "^0.12.31",
+    "karma-chrome-launcher": "^0.1.7",
+    "karma-jasmine": "^0.3.5",
+    "karma-phantomjs-launcher": "^0.1.4",
+    "protractor": "^2.0.0"
+  }
+}
```

Comment:
Does the app know that bower is a dependency? Would I be able to clone your project on a machine without a global installation of bower and know that I need to have it installed?